### PR TITLE
URP updates for Android build

### PR DESCRIPTION
**Pull request by fwild**
* added missing feature setting for AR camera in URP profile
* added network capability for android manifest
* set FXAA as anti-aliasing as TAA was not valid for Android
* added missing meta file msc.rsp.meta (for compiler flags already in the project)

This resolves the issue that the camera shows up as black on Android builds
and that it can reach the Moodle server again.

### CODE REVIEW

**Comment by fwild** [comment]
Resolves #1952 and #1953

File: Assets/msc.rsp.meta
Comment (BenediktHensen) [inline Assets/msc.rsp.meta:1]: We wanted to get rid of the msc.rsp file (see #1904) and so, the file, along with its meta file, were removed in an earlier pull request. However, some time after that, we ended up in a strange in-between stage where the msc.rsp file re-appeared but its meta file is still deleted. So, now Unity keeps adding a new meta file for the file on every new branch.

This doesn't block this pull request but we should ensure after merging this that we get back into a good state where the msc.rsp file is completely deleted.

File: Assets/MirageXR/Rendering/UniversalRenderPipelineGlobalSettings.asset
Comment (BenediktHensen) [inline Assets/MirageXR/Rendering/UniversalRenderPipelineGlobalSettings.asset:1]: Is Unity stripping away too many shader variants so that we need to disable this optimization?

In the future, we should look into methods to define exactly which shader variants are required so that we can re-enable stripping the unused variants.

**Review by BenediktHensen — APPROVED** [review]

File: Assets/MirageXR/Rendering/UniversalRenderPipelineGlobalSettings.asset
Comment (fwild) [reply Assets/MirageXR/Rendering/UniversalRenderPipelineGlobalSettings.asset:1]: good point - I had added this, as we lost the shadows AND the 3d model sketchfab imports. Not actually sure this is needed!

**Pull request merged**
